src/components/Shared/Filters/Filter.ts

```diff
@@ -0,0 +1,10 @@
+export type Filter = {
+  title: string;
+  type: string;
+  options: FilterOption[];
+};
+
+export type FilterOption = {
+  id: string | null;
+  name: string;
+};
```

Comment:
I started creating these as a generic class to share between ContactFilters and the future TaskFilters. I'm assuming they would have 2 different GraphQL types, but I wanted to be able to re-use components/types between the 2.

Reply:
I was thinking about that a little. It would be great if we could share GraphQL types too but that probably doesn't need to be addressed immediately. The `contactFilters` and `taskFilters` could both return a `FilterGroup` type if we can get all the fields to match up.

If we get that far, we could make a GraphQL fragment for all the fields needed by this filter component and use that fragment for the types here and use that fragment in whatever query is loading the filters.

Reply:
I'll have to take a look at GraphQL fragments, that might be a better immediate solution that will push us towards sharing this logic as the task and contact filter APIs are implemented